README.md

```diff
@@ -88,102 +93,45 @@ Additionally, for the development we use:
 
 > See the [Development](#development) section for details on how to install `mockery`.
 
-# Quick-Start Installation
+# Quick-Start installation
 
-We provide an installation script to automatically install and update the latest version of Trento.
+We provide installations scripts for to automatically install and update the latest version of Trento.
```

Comment:
I know that this is inherited, but in my opinion (and this came from our Doc team), we shouldn't use the `we` person here. I would re-write to something like: `The project provides installation scripts...`